### DISCUSSION TITLE
Add proxy support

### DISCUSSION THREAD

**Discussion by Alaneczek123123851** [NihadBadalov/RobloxAccountGeneratorSelenium#2] [Ideas]
Can you make an option to make a custom username? Example, Zionek_{Number from 1 to 99999}

**Comment by NihadBadalov** [comment]
@Alaneczek123123851 You can do that; Function on ln 50 **generateUserPass** should return an object:
```
return { "username": user, "password": password }
```

You can change the code so it returns *Zionek_{number}* and a password

**Reply by Alaneczek123123851** [reply]
> @Alaneczek123123851 You can do that; Function on ln 50 **generateUserPass** should return an object:
> 
> ```
> return { "username": user, "password": password }
> ```
> 
> You can change the code so it returns _Zionek_{number}_ and a password

I have question, Can you make captcha bypass?

**Reply by Alaneczek123123851** [reply]
and proxy

**Reply by NihadBadalov** [reply]
Not sure about captcha bypass, but this can probably be used with proxy

**Reply by NihadBadalov** [reply]
Either use a webproxy or check [something like this](https://stackoverflow.com/questions/17082425/running-selenium-webdriver-with-a-proxy-in-python)

**Reply by Alaneczek123123851** [reply]
You could write an ai that will define the images

**Reply by Alaneczek123123851** [reply]
So, Will you add proxy support?

**Reply by NihadBadalov** [reply]
I might

**Reply by Alaneczek123123851** [reply]
Can you help me with my python code? if yes then add on discord, Zionek#0001

**Comment by NihadBadalov** [comment]
Forgot about this a while ago.
Web proxies cannot be used with the rblx website since - I already forgot, but if I recall correctly - it has CORS  and/or something else that prevents requests from nonnative sources.
One could use a proxy; however, since that's _your_ choice, you'll have to google and find out by yourself. [This](https://stackoverflow.com/questions/17082425/running-selenium-webdriver-with-a-proxy-in-python) might be something you're looking for. Keep in mind that you will have to do this by yourself.
